src/Cortside.RestApiClient/IRestApiResponse.cs

```diff
@@ -0,0 +1,28 @@
+//using System;
```

Comment:
intentionally commented out?

Reply:
for now, yes.  but I think i am going to have to/want to implement that at some point.